src/e2e-test/features/DesignTimeValidation.feature

```diff
@@ -0,0 +1,199 @@
+    And Click on the "Next" button in replication to navigate
+    Then Enter input plugin property: "datasetName" with value: "dataset"
+    Then Replace input plugin property: "project" with value: "projectId"
+    And Click on the "Next" button in replication to navigate
+    Then Verify that the Plugin is displaying an error message: "errorMessageInvalidUser"
+
+  Scenario: To verify validation message when user provides invalid password
+    Given Open DataFusion Project with replication to configure replication job
+    When Enter input plugin property: "name" with pipelineName
+    And Click on the "Next" button in replication to navigate
+    And Select Source plugin: "Oracle" from the replication plugins list
+    Then Select radio button plugin property: "connectivityMethod" with value: "private-connectivity"
+    Then Enter input plugin property: "privateConnectionName" with value: "privateConnectionName"
+    Then Replace input plugin property: "host" with value: "oracleHost" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "oraclePort" for Credentials and Authorization related fields
+    Then Replace input plugin property: "sid" with value: "oracleDatabaseName"
+    Then Replace input plugin property: "user" with value: "oracleInvalidUser" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "oracleInvalidPassword" for Credentials and Authorization related fields
+    And Click on the "Next" button in replication to navigate
+    Then Enter input plugin property: "datasetName" with value: "dataset"
+    Then Replace input plugin property: "project" with value: "projectId"
+    And Click on the "Next" button in replication to navigate
+    Then Verify that the Plugin is displaying an error message: "errorMessageInvalidPassword"
+
+  Scenario: To verify validation message when macro enabled for password field
+    Given Open DataFusion Project with replication to configure replication job
+    When Enter input plugin property: "name" with pipelineName
+    And Click on the "Next" button in replication to navigate
+    And Select Source plugin: "Oracle" from the replication plugins list
+    Then Select radio button plugin property: "connectivityMethod" with value: "private-connectivity"
+    Then Enter input plugin property: "privateConnectionName" with value: "privateConnectionName"
+    Then Replace input plugin property: "host" with value: "oracleHost" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "oraclePort" for Credentials and Authorization related fields
+    Then Replace input plugin property: "sid" with value: "oracleDatabaseName"
+    Then Replace input plugin property: "user" with value: "oracleUsername" for Credentials and Authorization related fields
+    Then Click on the Macro button of Property: "password" and set the value to: "Password"
```

Comment:
Nit: 'Password' -> 'oraclePassword' like in other cases

Reply:
Change done.